pep/translations/en-us.json

```diff
@@ -61,6 +61,14 @@
             "reportDataError": "Report a Data Error",
             "preferences": "Help Preferences",
             "provideFeedback": "Provide Feedback"
+        },
+        "admin": {
+            "common": "Common",
+            "languages": {
+                "enUs": "English",
+                "esEs": "Spanish",
+                "frFr": "French"
+            }
```

Comment:
just labeling the nav item (and page title) "Common" seems a little vague, what about "Common Settings"?

additionally, it might be nice to below the dropdown divider have a dropdown-title (i think thats the bootstrap class) heading that is "Language Settings" or "Content Settings".